sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java

```diff
@@ -84,6 +85,7 @@
  * not interesting.
  */
 @SuppressWarnings("rawtypes")
+@Internal
```

Comment:
I believe the package doc states that all these util/* classes are internal but it might be wise to mark them `@Internal` anyways.

Reply:
It does not but I am going to add that to the package. Probably worth to do that too for the schema package with the Experimental tag too.